homeassistant/components/switcher_kis/utils.py

```diff
@@ -42,3 +43,13 @@ def on_device_data_callback(device: SwitcherBase) -> None:
 def get_breeze_remote_manager(hass: HomeAssistant) -> SwitcherBreezeRemoteManager:
     """Get Switcher Breeze remote manager."""
     return SwitcherBreezeRemoteManager()
+
+
+async def validate_input(username: str, token: str) -> bool:
+    """Validate token by specifying username and token."""
+    token_is_valid = await validate_token(username, token)
+    if token_is_valid:
+        _LOGGER.info("Token is valid")
+        return True
+    _LOGGER.info("Token is invalid")
+    return False
```

Comment:
Why do we need a new method here which just return the same data from `validate_token`? please use it directly where needed. Also avoid using logger in `info` level. 
Here there is no need for logging the result as it will be visible to the user when he enter the token.